src/components/TransactionItem/TransactionItem.jsx

```diff
@@ -0,0 +1,55 @@
+import { DeleteIcon, EditIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Flex,
+  HStack,
+  IconButton,
+  Tooltip,
+  useColorModeValue
+} from '@chakra-ui/react';
+import { format } from 'date-fns';
+
+import { NoteIcon } from '@/assets';
+
+export const TransactionItem = ({ transaction, onEdit, onDelete }) => {
+  const bgMain = useColorModeValue('orange.50', 'teal.700');
```

Comment:
use that variable from theme hook

Reply:
You can take a look at Preload component if you wanna see some example how to use that hook